subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/billing/RealBillingClientAdapter.kt

```diff
@@ -146,6 +149,43 @@ class RealBillingClientAdapter @Inject constructor(
         }
     }
 
+    override suspend fun queryPurchases(): QueryPurchasesResult = withContext(coroutineDispatchers.io()) {
```

Comment:
nit: Unless I'm missing something, this method doesn't do any blocking I/O operations, so switching dispatcher is unnecessary.